components/database/mongodb-40/mongodb40.p5m

```diff
@@ -21,7 +21,7 @@ set name=info.source-url value="$(COMPONENT_ARCHIVE_URL)"
 set name=info.upstream-url value="$(COMPONENT_PROJECT_URL)"
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
-license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)"
+license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)" must-accept=true must-display=true
```

Comment:
Please, do the same for mongodb40-service.p5m.

Reply:
@Mno-hime I'm just interested, should we add both or must-display is enough?

Reply:
I am fine with sole `must-display=true`. My point is to go an extra mile with this license, so users are safe. Hopefully with the license displayed, they figure out this time it's different and "Server Side" license on their screen rings some bells.